app/components/view-epics/macro.njk

```diff
@@ -0,0 +1,99 @@
+{% from 'components/tag/macro.njk' import tag %}
+{% from 'components/view-data-title/macro.njk' import viewDataTitle %}
+{% from 'components/view-epic-list/macro.njk' import viewEpicList %}
+
+{% macro viewEpics(params) %}
+  {% if params.data %}
+    <div data-test-id="{{ params.dataTestId }}" class="bc-c-epics {{ params.classes }}">
+      <div class="nhsuk-grid-row">
+        {% if params.data.must %}
+          <div class="must-epics nhsuk-grid-column-one-half">
+            <div>
+              {{ tag({
+                dataTestId: "example",
+                text: "Must Epics",
+                classes: "bc-c-tag-outline"
+              }) }}
+            </div>
+            {% if params.data.must['met'] %}
+              <div class="must-met-epics">
+                <div class="nhsuk-u-padding-top-5">
+                  {{ viewDataTitle({
+                    title: "Must Epics that have been met"
+                  }) }}
+                </div>
+                <div>
+                  {{ viewEpicList({
+                    dataTestId: "some-test-identifier",
+                    data: params.data.must['met'],
+                    type: "met"
+                  }) }}
+                </div>
+              </div>
+            {% endif %}
+            {% if params.data.must['not-met'] %}
+              <div class="must-not-met-epics">
```

Comment:
this class name needs to be consistent with the naming convention or if it is not used the whole div can be removed